pydm/data_plugins/plugin.py

```diff
@@ -55,14 +55,14 @@ def add_listener(self, channel):
             except TypeError:
                 pass
             try:
-                self.new_value_signal[ndarray].connect(channel.value_slot, Qt.QueuedConnection)
+                self.new_value_signal[bool].connect(channel.value_slot, Qt.QueuedConnection)
             except TypeError:
                 pass
             try:
-                self.new_value_signal[bool].connect(channel.value_slot, Qt.QueuedConnection)
+                self.new_value_signal[object].connect(channel.value_slot, Qt.QueuedConnection)
```

Comment:
it appears to still work for ndarray's, it might be possible to only have object and not any other types specified? because object can pass any data type with it.